### PR TITLE
Improve doc comments

### PR DESCRIPTION
**Pull request by petea**

### CODE REVIEW

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h [hunk 1]
```diff
@@ -81,7 +80,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 #if TARGET_OS_IOS || TARGET_OS_MACCATALYST
 
-/// Starts an interactive consent flow on iOS to add scopes to the current user's grants.
+/// Starts an interactive consent flow on iOS to add to the user's `grantedScopes`.
```
Comment (Alex-4-Git) [inline GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h:83]: add new scopes to?

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h [hunk 2]
```diff
@@ -54,13 +54,13 @@ typedef NS_ERROR_ENUM(kGIDSignInErrorDomain, GIDSignInErrorCode) {
 /// Represents a completion block that takes an error if the operation was unsuccessful.
 typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
 
-/// This class signs the user in with Google.
+/// This class is used to sign-in users with their Google account and manage their session.
```
Comment (Alex-4-Git) [inline GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h:57]: used to sign in..

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h [hunk 3]
```diff
@@ -84,23 +84,22 @@ typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
...
-/// Attempts to restore a previously authenticated user without interaction.
+/// Attempts to restore a previous user sign-in without interaction.
 ///
 /// @param completion The block that is called on completion.  This block will be called asynchronously
 ///     on the main queue.
 - (void)restorePreviousSignInWithCompletion:(nullable void (^)(GIDGoogleUser *_Nullable user,
                                                                NSError *_Nullable error))completion;
 
-/// Marks current user as being in the signed out state.
+/// Signs out the `currentUser`, removing them from the keychain.
```
Comment (Alex-4-Git) [inline GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h:99]: it instead of them?

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h [hunk 4]
```diff
@@ -84,23 +84,22 @@ typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
 /// @return `YES` if `GIDSignIn` handled this URL.
 - (BOOL)handleURL:(NSURL *)url;
 
-/// Checks if there is a previously authenticated user saved in keychain.
+/// Checks if there is a previous user sign-in saved in keychain.
```
Comment (Alex-4-Git) [inline GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h:87]: signed-in user?

**Review by mdmathias — APPROVED** [review]

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h (hunk 4 above)
Comment (petea) [reply GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h:87]: It's a little awkward, but I'm trying to be consistent with the method naming and the docs for `restorePreviousSignInWithCompletion:`.

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h (hunk 3 above)
Comment (petea) [reply GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h:99]: Updated.

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h (hunk 2 above)
Comment (petea) [reply GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h:57]: Good catch.  Updated.

File: GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h (hunk 1 above)
Comment (petea) [reply GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h:83]: I like it.  Updated.

**Review by Alex-4-Git — APPROVED** [review]

**Pull request merged**